collections/ansible_collections/purestorage/flashblade/plugins/modules/purefb_policy.py

```diff
@@ -1691,7 +1692,7 @@ def main():
                 )[0]
             except AttributeError:
                 new_policy = None
-        if policy and state == "present":
+        if policy and state == "present" and not module.params["rename"]
```

Comment:
Missing colon.
Can you also run the change against `black` to ensure it passes the lint tests

Reply:
Corrected and "black" lint check is passed in local